content/tracing/product_specs/distributed_tracing.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Getting started: Distributed Tracing "
+kind: documentation
+---
+
+If you have read the [first example of tracing](/tracing) and you want to understand deeper how tracing works, let's take the following example which represents a simple API **thinker-api** and a micro-service behind it **thinker-microservice**. When the API receives a request with the correct *subject* parameter, it responds with a *thought*, otherwise, it responds with an error:
```

Comment:
"want to more deeply understand how tracing works"

Reply:
👌